File: neumorphism/src/main/java/soup/neumorphism/internal/blur/BlurFactor.java
package soup.neumorphism.internal.blur;

import android.graphics.Color;

public class BlurFactor {

    public static final int DEFAULT_RADIUS = 25;
    public static final int DEFAULT_SAMPLING = 1;

    public int width;
    public int height;
    public int radius = DEFAULT_RADIUS;
    public int sampling = DEFAULT_SAMPLING;
    public int color = Color.TRANSPARENT;
}

File: neumorphism/src/main/java/soup/neumorphism/internal/blur/BlurFactor.kt
package soup.neumorphism.internal.blur

import android.graphics.Color

internal data class BlurFactor(
    val width: Int,
    val height: Int,
    val radius: Int = MAX_RADIUS,
    val sampling: Int = DEFAULT_SAMPLING,
    val color: Int = Color.TRANSPARENT
) {
    companion object {
        const val MAX_RADIUS = 25
        const val DEFAULT_SAMPLING = 1
    }
}
